gfx/NeoDays/pngs_tiles_10x10/items/adobe_mortar_0.json

```
@@ -0,0 +1 @@
+{"id": "mortar_adobe", "fg": ["3089_adobe_mortar_0"], "bg": []}
```

```suggestion
{"id": "mortar_adobe", "fg": "3089_adobe_mortar_0"}
```

were these files produced by some script? You can turn arrays into the string when they have only one element and drop empty `bg` values completely

---

Files were produced by carefully edited copypasta - I can leave out empty "bg" information in the future.  My apologies, my most significant programming experience was ZZT-OOP like 25 years ago and I'm always reluctant to make changes to things that might end up being necessary.